corehq/apps/style/static/style/js/hq.helpers.js

```diff
@@ -122,3 +122,9 @@ $.fn.enableButton = function () {
 };
 
 
+$.fn.koApplyBindings = function (context) {
+    if (!this.length) {
+        throw "No element passed to koApplyBindings";
```

Comment:
[Best practice](http://stackoverflow.com/a/27501348/10840) is to `throw new Error("...")`. Browsers like Chrome will display a stack trace, but other tools like phantomjs will not, which can make debugging more difficult.